packages/core/template/portfolio/contents/project1.md

```diff
@@ -0,0 +1,23 @@
+<frontmatter>
+  title: Project 1
+</frontmatter>
+
+<br>
+
+### Project 1
+
+<div class="card mb-3">
+  <h3 class="card-header">Name of project</h3>
+  <div class="card-body">
+    <h5 class="card-title">Short description of project</h5>
```

Comment:
Don't think this needs to be a header. It doesn't look like a header either haha